src/packaging/resource/cassandra-reaper-cassandra-sidecar.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2015-2017 Spotify AB
+# Copyright 2016-2019 The Last Pickle Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Cassandra Reaper Configuration Example.
+# See a bit more complete example in:
+# src/server/src/test/resources/cassandra-reaper.yaml
+segmentCountPerNode: 16
+repairParallelism: DATACENTER_AWARE
+repairIntensity: 0.9
+scheduleDaysBetween: 7
+repairRunThreadCount: 15
+hangingRepairTimeoutMins: 30
+storageType: cassandra
+enableCrossOrigin: true
+incrementalRepair: false
+blacklistTwcsTables: false
+enableDynamicSeedList: true
+repairManagerSchedulingIntervalSeconds: 10
+activateQueryLogger: false
+jmxConnectionTimeoutInSeconds: 5
+useAddressTranslator: false
+# purgeRecordsAfterInDays: 30
+# numberOfRunsToKeepPerUnit: 10
+enableConcurrentMigrations: true
+
+# datacenterAvailability has three possible values: ALL | LOCAL | EACH
```

Comment:
trivial: this comment now needs to include info about `SIDECAR` (same goes for all our *.yaml files)

Reply:
yes, done 👍